vendor/brave-ios/Ads/BATBraveAds.h

```diff
@@ -140,18 +121,31 @@ NS_SWIFT_NAME(BraveAds)
 - (void)reportTabClosedWithTabId:(NSInteger)tabId NS_SWIFT_NAME(reportTabClosed(tabId:));
 
 /// Report that an ad notification event type was triggered for a given id
-- (void)reportAdNotificationEvent:(NSString *)uuid
-                        eventType:(BATAdNotificationEventType)eventType;
+- (void)reportAdNotificationEvent:(NSString*)uuid
+                        eventType:(BATBraveAdsAdNotificationEventType)eventType;
+
+/// Get inline content ad for the given dimensions
+- (void)inlineContentAdsWithDimensions:(NSString*)dimensions
+                            completion:(void (^)(BOOL success,
+                                                 BATInlineContentAd*)))completion
+    NS_SWIFT_NAME(inlineContentAdsWithDimensions(dimensions:completion:));
```

Comment:
Sorry one last thing:

```suggestion
    NS_SWIFT_NAME(inlineContentAds(dimensions:completion:));
```

Otherwise naming and args are duped in Swift

Reply:
Fixed